lib/addons/aws-privateca-issuer/index.ts

```diff
@@ -0,0 +1,95 @@
+// lib/kubevious_addon.ts
+import { Construct } from 'constructs';
+import { ManagedPolicy } from "aws-cdk-lib/aws-iam";
+import merge from "ts-deepmerge";
+import { HelmAddOn, HelmAddOnUserProps, HelmAddOnProps } from "../helm-addon";
+import { dependable } from '../../utils';
+import { setPath } from '../../utils/object-utils';
+import { ClusterInfo, Values } from "../../spi";
+import { createNamespace } from "../../utils";
+/**
+ * User provided options for the Helm Chart
+ */
+export interface AWSPrivateCAIssuerAddonProps extends HelmAddOnUserProps {
+    /**
+     * Specifies whether a service account should be created
+     */
+    createServiceAccount?: boolean;

```

Comment:
it sounds like it is a similar issue as in cert-manager. We should not approach it as a helm chart, this is an add-on. So createServiceAccount - true should indicate whether CDK should create a service account and false is to delegate this feature to the helm chart. 
Let's make it consistent with the cert-manager and other add-ons.

Reply:
yes.. agree.. made it same like keda, If user provides the Roles, then cdk will create service account